src/transformers/pipelines/mask_generation.py

```diff
@@ -55,15 +55,21 @@ class MaskGenerationPipeline(ChunkPipeline):
         tokenizer ([`PreTrainedTokenizer`]):
             The tokenizer that will be used by the pipeline to encode data for the model. This object inherits from
             [`PreTrainedTokenizer`].
-        feature_extractor ([`SequenceFeatureExtractor`]):
-            The feature extractor that will be used by the pipeline to encode the input.
+        image_processor ([`BaseImageProcessor`]):
+            The image processor that will be used by the pipeline to encode data for the model. This object inherits from
+            [`BaseImageProcessor`].
```

Comment:
Will another `image_processor` snippet be appended with `build_pipeline_init_args`? Same question for `device` below.

Reply:
Good point. Yes, they would be. I've removed all the args here and added additional ones to the `add_end_docstring` decorator